service-tests/bugzilla.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const bzBugStatus = Joi.equal(
+  'unconfirmed',
+  'new',
+  'assigned',
+  'fixed',
+  'invalid',
+  'won\'t fix',
+  'duplicate',
+  'works for me',
+  'incomplete'
+);
+
+const t = new ServiceTester({ id: 'bugzilla', title: 'Bugzilla' });
+module.exports = t;
+
+t.create('Bugzilla valid bug status')
+  .get('/996038.json')
+  .inspectJSON()
+  .expectJSONTypes(Joi.object().keys({
+    name: 'Bug 996038',
+    value: bzBugStatus
+  }));
+
+t.create('Bugzilla invalid bug status')
+  .get('/83548978974387943879.json')
+  .expectJSON({ name: 'Bug 83548978974387943879', value: 'unknown' });
+
+t.create('Bugzilla failed request bug status')
+  .get('/996038.json')
+  .networkOff()
+  .expectJSON({ name: 'Bug 996038', value: 'unknown' });
```

Comment:
could you also add 1 more test for an invalid JSON response.
[example](https://github.com/badges/shields/blob/master/service-tests/cdnjs.js#L28-L34)

Reply:
I made all the changes except this one, for some reason I couldn't get this to work as nock kept throwing `NetConnectNotAllowedError` errors.